test/unit/query/test-error-event.js

```diff
@@ -7,11 +7,13 @@ var server = common.createFakeServer();
 server.listen(common.fakeServerPort, function (err) {
   assert.ifError(err);
 
-  var query = connection.query('INVALID SQL');
+  var sql = 'INVALID SQL';
```

Comment:
Hi @magicdawn sorry I didn't provide more content to my comment above :) So "DRY" is an acronym, but essentially my comment is just saying to remove the `sql` variable and just use the string `'INVALID SQL'` in the file twice.